java/src/server/commands/moves/RollNumberCommand.java

```diff
@@ -9,21 +13,32 @@
  * @author Joel Bradley
  */
 public class RollNumberCommand implements ICommand {
+    private int gameId;
+    private int value;
+    private int playerIndex;
+    private IFacade facade;
 
     /**
      * Constructor
      */
-    public RollNumberCommand() {
-
+    public RollNumberCommand(IFacade facade, RollNumberDTO roll) {
+        this.facade = facade;
```

Comment:
So instead of storing `gameId`, `value`, and `playerIndex` you can just store `dto` or `rollDTO` or something.  Then down lower in `facade.rollNumber()` you can pass in `dto.get` whatever you need.